model/config.go

```diff
@@ -19,6 +19,8 @@ package model
 
 import (
 	"time"
+
+	"github.com/elastic/apm-server/model/metadata"
```

Comment:
I find the filename `config.go` confusing, as it defines the `Input` struct holding real event data. Can you rename to `model/input.go`?

Reply:
Yes, good call. I thought about it and didn't because I intended to move this to another package altogether - but I'll rename it anyway, as that might not happen for a while.